src/main/java/org/folio/fqm/service/FqmTenantService.java

```diff
@@ -20,6 +20,15 @@ public FqmTenantService(JdbcTemplate jdbcTemplate, FolioExecutionContext context
   @Override
   public synchronized void createOrUpdateTenant(TenantAttributes tenantAttributes) {
     this.folioSpringLiquibase.setChangeLogParameters(Map.of("tenant_id", this.context.getTenantId()));
+
+    // Run all of the regular DB migration scripts (those that aren't marked as "slow")
+    this.folioSpringLiquibase.setContexts("!slow");
     super.createOrUpdateTenant(tenantAttributes);
+
+    // Run all of the slow DB migration scripts in a separate thread
+    this.folioSpringLiquibase.setContexts("slow");
+    var slowRun = new Thread(() -> super.createOrUpdateTenant(tenantAttributes));
+    slowRun.setDaemon(true);
```

Comment:
Just to be safe, I set it as a daemon thread, so that a slow-running query doesn't prevent the JVM from shutting down